File: moneylog_spring/src/main/java/bitc/fullstack503/moneylog_spring/dto/ExpenseLogDTO.java
package bitc.fullstack503.moneylog_spring.dto;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

import java.time.LocalDateTime;
@Data
public class ExpenseLogDTO
{
  @JsonProperty ("expenseLogSeq")
  private int expenseLogSeq;
  @JsonProperty ("memberId")
  private String memberId;
  @JsonProperty ("expenseMoney")
  private int expenseMoney;
  @JsonProperty ("expenseDate")
  private LocalDateTime expenseDate;
  @JsonProperty ("expenseCate")
  private String expenseCate;
  @JsonProperty ("expenseMemo")
  private String expenseMemo;
  @JsonProperty ("paymentOption")
  private String paymentOption;
  @JsonProperty ("expenseUse")
  private String expenseUse;
}

File: moneylog_app/app/src/main/java/bitc/example/app/DTO/ExpenseLogDTO.kt
package bitc.example.app.dto

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class ExpenseLogDTO(
  @SerializedName("expenseLogSeq")
  var expenseLogSeq: Int? = null,
  @SerializedName("memberId")
  var memberId: String? = null,
  @SerializedName("expense")
  var expense: Int? = null,
  @SerializedName("expenseDate")
  var expenseDate: LocalDateTime? = null,
  @SerializedName("expenseCate")
  var expenseCate: String? = null,
  @SerializedName("expenseMemo")
  var expenseMemo: String? = null,
  @SerializedName("expenseSource")
  var expenseSource: String? = null
)
